### PR TITLE
Dishes have a category

### PR DESCRIPTION
**Pull request by fabianlundgren**
Corrected the branching

@AmberWilkie

### CODE REVIEW

**Comment by AmberWilkie** [comment]
I renamed this feature. I would like your titles to be something more like this - here and in the cards as well. Your user stories should go into the *descriptions* of your cards.

File: lib/views/menu.erb [hunk 1]
```diff
@@ -1,5 +1,21 @@
 <h1>Our Menu</h1>
 
-  <% @dishes.each do | dish | %>
-  <p><%= dish.name %>, <%= dish.price %></p>
-  <% end %>
+<h2>Categories</h2>
```
Comment (AmberWilkie) [inline lib/views/menu.erb]: We probably don't need to see Categories here, right? From a design perspective, does it make sense?

File: lib/views/menu.erb [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+
+   <% @main_course.each do | dish | %>
+   <p><%= dish.name %>, <%= dish.price %></p>
+   <% end %>
+
+<h3>Desserts</h3>
+
+   <% @dessert.each do | dish | %>
+   <p><%= dish.name %>, <%= dish.price %></p>
+   <% end %>
```
Comment (AmberWilkie) [inline lib/views/menu.erb]: You should indent this file like you would regular ruby code - that is, I want to see the lines with `<p>` indented two spaces so they are "inside" the `do / end` blocks.

File: spec/categories_spec.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+require './lib/models/dish.rb'
+
+describe Dish do
+
+  it { is_expected.to have_property :category }
+
+
+end
```
Comment (AmberWilkie) [inline spec/categories_spec.rb]: Instead of making a whole file for this, you can add this line to the `dish_spec`.

**Review by AmberWilkie — COMMENTED** [review]
Looks good, needs a few changes.

**Review by diraulo — COMMENTED** [review]
You don't create dishes in your cucumber tests with categories assigned to them. How do you actually know that this is working as it should?

From a unit test point of view, the dish model sure works as expected. 

In my opinion, on the acceptance test side, I'd expect you to create some dishes and test that they're listed on the menu page accordingly. That no dish that belongs to the starter's category, for instance, appears in another group.

**Comment by casperjureskog** [comment]
hi Raoul i think we got a bit confused, because we hade a check in an earlier PR that we where waiting for. we will fix it :-) not relay use to github so far.

File: lib/controller.rb [hunk 4]
```diff
@@ -56,6 +56,9 @@ class SlowFood < Sinatra::Base
 
   get '/menu' do
     @dishes = Dish.all
+    @starter = Dish.all(category: 'starter')
```
Comment (tochman) [inline lib/controller.rb]: So you are no longer using `@dishes` in your view, right? Please remove unused variables.

**Comment by tochman** [comment]
Apart from an unused variable this seems ready to be merged.

**Pull request merged**